File: LeetCode/java/2678_Number_of_Senior_Citizens.java
class Solution {
    public int countSeniors(String[] details) {
        int answer = 0;

        for (String detail : details) {
            StringBuilder age = new StringBuilder();

            age.append(detail.charAt(11));
            age.append(detail.charAt(12));

            if (Integer.parseInt(age.toString()) > 60) {
                answer++;
            }
        }

        return answer;
    }
}

File: LeetCode/cpp/2678_Number_of_Senior_Citizens.cpp
class Solution {
public:
	int countSeniors(vector<string>& details) {
		int answer = 0;

		for (string detail : details) {
			string age;

			age += detail[11];
			age += detail[12];

			if (stoi(age) > 60) {
				answer++;
			}
		}

		return answer;
	}
};